crates/matrix-sdk-ui/src/timeline/mod.rs

```diff
@@ -414,6 +416,68 @@ impl Timeline {
         Ok(())
     }
 
+    /// Send a reply to the given event.
+    ///
+    /// Currently only supports `m.room.message` events whose event ID is known
+    /// and will log an error and do nothing for other events. Please check
+    /// [`EventTimelineItem::can_be_edited`] before calling this.
+    ///
+    /// # Arguments
+    ///
+    /// * `content` - The content of the reply
+    ///
+    /// * `reply_item` - The event item you want to reply to
+    ///
+    /// * `forward_thread` - Usually `Yes`, unless you explicitly want to the
+    ///   reply to show up in the main timeline even though the `reply_item` is
+    ///   part of a thread
+    ///
+    /// * `add_mentions` - Set to `Yes` if the `mentions` of `content` are
+    ///   propagated according to user intent, `No` otherwise
```

Comment:
Something wrong with this comment :grin: The function takes only 2 non-self parameters, and they're not called that.